cirq/google/engine/calibration_test.py

```diff
@@ -163,11 +163,13 @@ def test_calibration_heatmap():
     figure = mpl.figure.Figure()
     axes = figure.add_subplot(111)
     heatmap.plot(axes)
+    assert axes.get_title() == 'T1'
 
     heatmap = calibration.heatmap('xeb')
     figure = mpl.figure.Figure()
     axes = figure.add_subplot(999)
     heatmap.plot(axes)
+    assert axes.get_title() == 'Xeb'
```

Comment:
can you add a test for the underscore replacement as well, please?

Reply:
done.